deploy_docs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# create local env
+conda create -n python36 python=3.6 --yes
+
+source activate python36
+
+conda install -c conda-forge doctr --yes
+
+doctr deploy --no-require-master . --built-docs build/sphinx/html 
```

Comment:
why `no-require-master` - what does that do?

Reply:
It allows docs to be generated even from non master branches

Reply:
That would re-generate the main documentation even for PRs - which is not what we want. We want to review the documentation before it goes live.